### ISSUE TITLE
train problem

### ISSUE THREAD

**Issue by betterhalfwzm** [zylo117/Yet-Another-EfficientDet-Pytorch#96]
I'm sorry to bother you again. When i train custom dataset, After several eopch, the score reached the highest level, and then began to decline，how can i solve it?
Example of pictures：
![image](https://user-images.githubusercontent.com/20449990/79408070-04933180-7fcd-11ea-8a4d-ba4f8fa70928.png)
![image](https://user-images.githubusercontent.com/20449990/79408087-0eb53000-7fcd-11ea-8e3c-5234838cf186.png)

**Comment by zylo117** [comment]
it's overfitting

**Comment by betterhalfwzm** [comment]
> it's overfitting
How to solve over fitting in your code? And there are a lot of negative samples in my data set. Does this affect? How should I adjust the code to add negative sample training?

**Comment by betterhalfwzm** [comment]
> it's overfitting

How to solve over fitting in your code? And there are a lot of negative samples in my data set. Does this affect? How should I adjust the code to add negative sample training?

**Comment by heitorrapela** [comment]
@betterhalfwzm try to add some early stopping and save the best model on validation loss (split the data in train / val /  test ).
